readme.md

```diff
@@ -37,6 +37,8 @@ const cpy = require('cpy');
 
 Returns a `Promise<string[]>` with the destination file paths.
 
+If the file does not exist, an error will be thrown (does not apply to globs).
+
```

Comment:
You need to update https://github.com/sindresorhus/cpy/blob/master/index.d.ts too.